style.css

```diff
@@ -1,34 +1,104 @@
 * {
   background-color: #fff;
-  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 18px;
 }
 
-.center {
+.container-nav-menu {
   display: flex;
-  flex-direction: column;
-  justify-content: center;
+  justify-content: space-around;
   align-items: center;
-  padding-top: 60px;
-}
-
-.button {
   border: 2px solid black;
-  width: 60px;
-  float: right;
 }
 
 .container {
+  padding: 5px;
   border: 2px solid black;
-  width: 50%;
+}
+
+.logo {
+  font-size: 40px;
+  font-weight: bold;
+}
+
+.menu__items {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.menu__items li {
+  list-style-type: none;
+  padding: 20px;
+}
+
+.middle-link {
+  border-left: 2px solid black;
+  border-right: 2px solid black;
+  padding: 10px;
+}
+
+.width-height {
+  width: 100%;
+}
+
+.head {
+  text-align: center;
+  font-size: 30px;
+}
+
+.list-list {
+  padding-top: 10px;
 }
 
 .displayRow {
   display: flex;
   justify-content: space-between;
-  align-items: center;
-  padding: 5px;
 }
 
 .displayRow ul li {
   list-style-type: none;
 }
+
+.add-form {
+  display: none;
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.contact-info {
+  display: none;
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.button {
```

Comment:
- [ ] [OPTIONAL]
In addition, I believe we can improve upon the UX of the nav area by ensuring the menu links behaves like clickable elements when the mouse hovers over them. We can achieve this by adding the CSS attribute `cursor: pointer`